Assets/MixedRealityToolkit-SDK/Features/UX/Interactable/Scenes/ButtonTest.unity

```diff
@@ -0,0 +1,973 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Test scenes shouldn't be in the SDK

Reply:
For me this is still a work in progress until we get all the functionality, like down and up on HoloLens working. I agree with your other comment below about moving the scene to examples. I may end up replacing it at some point or at least repurposing it to contain all the button components.